src/payment/strategies/ppsdk/initializers/none.ts

```diff
@@ -0,0 +1,23 @@
+import { createRequestSender } from '@bigcommerce/request-sender';
+import { isEqual } from 'lodash';
+
+import { PPSDKPaymentMethod  } from '../../../ppsdk-payment-method';
+import { PaymentProcessor } from '../ppsdk-payment-processor';
+import { PaymentsAPIResponse } from '../ppsdk-payments-api-response';
+import { stepHandler } from '../step-handler';
+
+interface None {
+    type: 'NONE';
+}
+
+export const isNone = (unknownStrategy: unknown): unknownStrategy is None =>
+    isEqual(unknownStrategy, { type: 'NONE' });
+
+export const initializeNone = (paymentMethod: PPSDKPaymentMethod): PaymentProcessor => {
```

Comment:
`stepHandler` will be used by each different initialisation strategy - it will consume the polymorphic `/payments` API responses and turn them into necessary action (redirecting, portraying success/failure etc).

Let me know if it makes more sense now things have been reorganised 👍